server/src/envelope.rs

```diff
@@ -449,11 +473,13 @@ impl Envelope {
     }
 
     /// Returns the number of items in this envelope.
+    #[allow(dead_code)]
```

Comment:
Can we consider removing this? If it's really unused there's no point in keeping it.